### ISSUE TITLE
Open collision editor from the tile object context menu

### ISSUE THREAD

**Issue by T1mL3arn** [bjorn/tiled#977]
Now to change the collision layer for the tile image is need to find that image in the tileset. Sometimes it may be a problem (when the tileset contains a lot of images). But may be new context menu element will be a good feature - something like this: Rightclick on the tile object - > Select "Show collision layer in the collision editor" -> the collision editor window for the tile image is open!

**Comment by bjorn** [comment]
I think it is a good idea to provide quick access to the collision shape when clicking on a tile. Maybe it could even be done without a menu item, if we change the current tile depending on the selection. We'd need to make sure this isn't annoying in some cases.

**Comment by T1mL3arn** [comment]
It may be annoying if the tileset in the tileset panel will be switched when the tile object on the map is clicked.

**Comment by bjorn** [comment]
@T1mL3arn Right. Maybe we could just change the current tile, without also switching to its tileset.

**Comment by bjorn** [comment]
Not sure if you still need the functionality, but I expect the above change will make life much easier in this situation.

It does only change the animation and collision editors to edit the tile associated with the object. It does not change the selected tile in the tileset view. I didn't do that latter because I thought it might be annoying if you're inserting objects, and them temporarily switch to drag some existing object around, that it would then start inserting with another tile when you resume inserting.

**Comment by T1mL3arn** [comment]
Oh, thanks! But I not see how it works for last build (13-Sep-2015 15:41 on this moment).

**Comment by bjorn** [comment]
> Oh, thanks! But I not see how it works for last build (13-Sep-2015 15:41 on this moment).

Alas, I submitted this change on the 14th but the daily builds are currently broken. I'll try to fix it as soon as possible. In the meantime, there is build with this change (but without Python plugin) available at https://ci.appveyor.com/project/bjorn/tiled/build/job/sr7rg819u8ue9bii/artifacts

**Comment by bjorn** [comment]
Daily builds are working again as of change 586ce2390abdbe14316f08af2bec2d89e1116da5!

**Comment by bjorn** [comment]
This improvement was lost in Tiled 1.0 due to the separation of tileset editing from map editing, which was done to avoid confusion regarding the undo/redo and saving system. I'm not sure of a nice way to regain this functionality, but at least I'll reopen this issue so that we may remember to think of some solution to this. Suggestions welcome.
